apps/src/sites/studio/pages/maker/setup.js

```diff
@@ -111,15 +121,15 @@ const BoardSetupStatus = React.createClass({
      */
     function forEachLedInSequence(func, delay) {
       return new Promise(resolve => {
-        const leds = bc.prewiredComponents.colorLeds;
+        const leds = board.prewiredComponents_.colorLeds;
```

Comment:
Hmm, not sure it's a good idea to access this "private" variable outside of the module it was defined in... unless you mean this to be a ["friend"](http://en.cppreference.com/w/cpp/language/friend)?